files/en-us/web/css/anchor/index.md

```diff
@@ -110,18 +108,18 @@ The CSS {{glossary("inset properties")}} that accept an `anchor()` function as a
 
 When using an `anchor()` function inside an inset property value, the `<anchor-side>` parameter specified inside the `anchor()` function has to be compatible with the axis on which the inset property resides.
 
-In the case of physical inset properties and `<anchor-side>` values, this is relatively simple. `top: anchor(bottom)` is fine, for example, but `top: anchor(left)` is not valid. If `top: anchor(left, 50px)` were specified, the fallback value would be used, so `top` would get a computed value of `50px`. If no fallback is present, the inset property has no effect, as if it were set to `auto`.
+This means that physical `<anchor-side>` values can be used within the values of physical inset properties if axis direction of the `<anchor-side>` is the same as the physical property. In other words, the `top` and `bottom` sides are only valid within the `top` and `bottom` property values (or the `top` and `bottom` values within the `inset` shorthand), and `left` and `right` are only valid within `left` and `right` property values. For example, `top: anchor(bottom)` is fine, as they are both vertical, or y-axis values, but `top: anchor(left)` is not valid, as `left` is a horizontal, or x-axis value. If `top: anchor(left, 50px)` were specified, the fallback value would be used, so `top` would get a computed value of `50px`. If no fallback is present, the inset property has no effect, as if it were set to `auto`.
```

Comment:
it does actually have an effect: `auto` is in the cascade, overwriting previous values with the same specficity. Ex:

```css
.foo {
  bottom: anchor(bottom);
  bottom: anchor(left);
}
```
the value for `foo`'s `bottom` is `auto` -> that's an effect.